### PR TITLE
CreateMerchantPymtSessionAPI - Return Session data

### PR DESCRIPTION
**Pull request by mchopra-woolworths**

### CODE REVIEW

**Review by lohithv — APPROVED** [review]
LGTM

File: src/wpay-1.json [hunk 1]
```diff
@@ -24235,12 +24235,17 @@
...
-                    "paymentSessionId"
+                    "paymentSessionId",
+                    "paymentSession"
                   ],
                   "properties":{
                     "paymentSessionId":{
                       "description":"The ID of the new payment session",
-                      "type":"string"
+                      "type":"string",
+                      "deprecated": true
```
Comment (lohithv) [inline src/wpay-1.json:24245]: but is this really deprecated ? Or are we indicating it will become obsolete in the future ?

**Review by chpdivya — APPROVED** [review]

File: src/wpay-1.json (hunk 1 above)
Comment (mchopra-woolworths) [reply src/wpay-1.json:24245]: Its indication from future perspective. All consumers of this API should use payment session id from the payment session object.

**Comment by mchopra-woolworths** [comment]
Created another PR with latest master copy

**Pull request closed without merging**